net.sourceforge.electrip.Electrip.json

```
@@ -0,0 +1,46 @@
+{
+    "app-id": "net.sourceforge.electrip.Electrip",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+    "command": "electrip",
+    "finish-args": [
+        "--socket=x11",
```

you also need --share=ipc if you have --socket=x11 per http://docs.flatpak.org/en/latest/sandbox-permissions.html
```suggestion
        "--socket=x11",
        "--share=ipc",
```